services/ui-src/src/verbiage/pages/mcpar/mcpar-intro.ts

```diff
@@ -1,6 +1,51 @@
+import NavigationSectionsImage from "../../../assets/images/NavigationSections_2x.png";
+import NavigationSectionsSubmissionImage from "../../../assets/images/NavigationSectionsSubmission_2x.png";
+
 export default {
   intro: {
-    header: "MCPAR Intro",
-    body: "Placeholder page",
+    header: "Before you begin the MCPAR online form",
+  },
+  body: {
+    sections: [
+      {
+        header: "Start by adding all Medicaid managed care programs",
+        body: "You will need to submit one MCPAR for every managed care program in your state. For this MCPAR report, a managed care program is defined by distinct benefits and eligibility criteria articulated in a contract between states and the state’s managed care plans.",
+        widget: {
+          title: "Examples of managed care programs:",
+          descriptionList: [
+            "Health and Recovery Plans (Comprehensive MCO)",
+            "Dental Managed Care",
+          ],
+        },
+      },
```

Comment:
As noted above, add a key/value pair of `sectionNumber: 1,` instead of passing the number in directly in the code. Since the numbers are really text instead of "number" numbers it makes more sense to keep them here in this static text file.